cfn/prod-release.yml

```diff
@@ -212,8 +209,8 @@ Resources:
             {
               "Effect": "Allow",
               "Resource": [
-                "arn:aws:kms:*:658956600833:key/*",
-                "arn:aws:kms:*:658956600833:alias/*"
+                "arn:aws:kms:*:${AWS::AccountId}:key/*",
```

Comment:
Nope! `658956600833` has the keys you need to pass the tests. 
This statement grants the CodeBuild resource permission to call the key.